tests/cases/conformance/types/spread/objectSpread.ts

```diff
@@ -78,4 +78,6 @@ let computedAfter: { a: number, b: string, "at the end": number } =
 // shortcut syntax
 let a = 12;
 let shortCutted: { a: number, b: string } = { ...o, a }
+// non primitive
+let spreadNonPrimitve = { ...<object>{}}
```

Comment:
typo: Primitive
(and missing some semi-colons, not very important in tests though)